ui/components/MesheryPatterns/MesheryPatternCard.js

```diff
@@ -38,11 +38,14 @@ function MesheryPatternCard({
   }
   const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
   const [fullScreen, setFullScreen] = useState(false);
+  const [showCode,setShowCode]=useState(false);
```

Comment:
here also space after the comma, @subhradwip01 did you used no-verify the linter should format it automatically?

Reply:
i have not used no-verify